### PR TITLE
Print layout broken in Firefox

### PR DESCRIPTION
**Pull request by gagan67commerce**
### Fixed Issues (if relevant)

1. magento/magento2#12464: print-issue-was-coming-in-firefox

### Manual testing scenarios

1. Make the changes in the print.css file via the PR
2. Perform php bin/magento s:s:d 
3. Perform php bin/magento c:f
4. Navigate to homepage and Click on the 3 lines/dots on top left of browser
5. Click on Print
6. You will be able to view a better update of the pdf layout

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2?pullRequest=13043) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/magento2?pullRequest=13043) before we can accept your contribution.<br/>

**Comment by southerncomputer** [comment]
This looks bad in firefox 57.0.4 - only the first page of product/comparison gets printed then a blank page comes out.

Looks far better on chrome however! Proper!

**Comment by gagan67commerce** [comment]
@magento-cicd2  Do you suggest that I should again write the css for print.css for all browsers, or what do you recommend best.

**Comment by southerncomputer** [comment]
I'd recommend you at least try it with firefox to see what i mean. Only one page gets printed then cut-off! 
Perhaps try to resolve that one issue before pushing a change to current code!

**Comment by okorshenko** [comment]
@magento-engcom-team give me new test instance

**Comment by magento-engcom-team** [comment]
Hi @okorshenko, here is your [new test Magento instance](http://pr-13043.engcom.dev.magento.com)

**Comment by okorshenko** [comment]
@magento-engcom-team give me test instance

**Comment by magento-engcom-team** [comment]
Hi @okorshenko, here is your [test Magento instance](http://pr-13043.engcom.dev.magento.com).
Admin access: [admin panel](http://pr-13043.engcom.dev.magento.com/admin) .
Login: `admin` ; Password: `123123q`

**Comment by okorshenko** [comment]
@magento-engcom-team give me new test instance

**Comment by magento-engcom-team** [comment]
Hi @okorshenko. Thank you for your request. I'm working on test Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @okorshenko, here is your [new test Magento instance](http://pr-13043.engcom.dev.magento.com).
Admin access: [http://pr-13043.engcom.dev.magento.com/admin](http://pr-13043.engcom.dev.magento.com/admin) .
Login: `admin` ; Password: `123123q`

**Comment by okorshenko** [comment]
@magento-engcom-team give me new test instance

**Comment by magento-engcom-team** [comment]
Hi @okorshenko. Thank you for your request. I'm working on test Magento instance for you

**Comment by magento-engcom-team** [comment]
Hi @okorshenko, here is your [new Magento instance](http://pr-13043.engcom.dev.magento.com).
Admin access: [http://pr-13043.engcom.dev.magento.com/admin](http://pr-13043.engcom.dev.magento.com/admin)
Login: `admin` Password: `123123q`

**Comment by VladimirZaets** [comment]
Hi @gagan67commerce, thank you for collaboration.
The print view mode looks much better. But category page in print view mode in Firefox still looks broken. Will be great, if you fix category page too and create new one PR.

[CategoryFirefoxAfter.pdf](https://github.com/magento/magento2/files/1927415/CategoryFirefoxAfter.pdf)

File: app/design/frontend/Magento/blank/web/css/print.less [hunk 1]
```diff
@@ -1,135 +1,70 @@
 /**
- * Copyright © Magento, Inc. All rights reserved.
+ * Copyright © 2013-2017 Magento, Inc. All rights reserved.
```
Comment (VladimirZaets) [inline app/design/frontend/Magento/blank/web/css/print.less:2]: Hi @gagan67commerce , please update copyright following Magento 2.2 design.

File: app/design/frontend/Magento/blank/web/css/print.less [hunk 2]
```diff
@@ -1,135 +1,70 @@
...
  * See COPYING.txt for license details.
  */
 
 @import 'source/lib/_lib.less'; // Global lib
 @import 'source/_theme.less'; // Theme overrides
 @import 'source/_variables.less'; // Local theme variables
 
+@import 'styles-m.less';
```
Comment (VladimirZaets) [inline app/design/frontend/Magento/blank/web/css/print.less:10]: Why did you include style-m and style-l to print.less file?
This styles already exists at the page, and print.css file changed size from 1.5kb to 64kb. It's а lot for print.css,
we can't accept it. 
Please remove redundant imports.

**Comment by VladimirZaets** [comment]
Hi @gagan67commerce  , any updates?

**Comment by gagan67commerce** [comment]
Hi @VladimirZaets . I am looking into this will update once done.

File: app/design/frontend/Magento/blank/web/css/print.less (hunk 2 above)
Comment (gagan67commerce) [reply app/design/frontend/Magento/blank/web/css/print.less:10]: OK

**Comment by VladimirZaets** [comment]
@gagan67commerce thanks

**Comment by VladimirZaets** [comment]
Hi @gagan67commerce, do you have any updates, maybe you need some help?

**Comment by VladimirZaets** [comment]
Hi @gagan67commerce, I closed this PR. If you find the solution please reopen or create new one PR. Thanks.

**Pull request closed without merging**